test/machines.jl

```diff
@@ -27,6 +27,12 @@ rms(yhat, y) ≈ std(y)*sqrt(1 - 1/n)
 # test an unsupervised univariate case:
 mach = machine(UnivariateStandardizer(), float.(1:5))
 @test_logs (:info, r"Training") fit!(mach)
+@test isempty(params(mach))
+
+# test a frozen NodalMachine
+stand = machine(Standardizer(), source(DataFrame(x1=rand(10))))
```

Comment:
I'd like to work towards removing DataFrames as a dependency. You can instead use `source((x1=rand(10),))` here.